docs/release_notes.md

```diff
@@ -1,4 +1,6 @@
 # Release Notes
+## 0.1.3
```

Comment:
We didn't release 0.1.2, so we can leave this as 0.1.2 - especially since you didn't change semver in this pr